### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by asofmosq**

### CODE REVIEW

File: providencePipeline.wdl [hunk 1]
```diff
@@ -195,36 +199,49 @@ task bbMap {
...
 
-    #Align fastq files to hg38 & only keep unmapped
-
-    bowtie2 -p ~{threads} -x ~{reference} \
-    -1 ~{fastq1} -2 ~{fastq2} \
-    --un-conc-gz ~{sample}_host_removed_r%.fastq.gz | \
-    samtools view -b | \
-    samtools sort - -o ~{hostMappedBam_}
+    #index the reference
+    bwa index ~{reference}
```
Comment (apmasell) [inline providencePipeline.wdl:236]: Is this index going to be written into the index storage directory? That might be a permissions issue. Maybe symlink it to a local path and index that?

File: info_for_rmarkdown.pl [hunk 2]
```diff
@@ -0,0 +1,252 @@
+use strict;
```
Comment (apmasell) [inline info_for_rmarkdown.pl:1]: Please run perltidy on this for formatting.

**Review by apmasell — APPROVED** [review]

File: providencePipeline.wdl [hunk 3]
```diff
@@ -4,162 +4,155 @@ workflow providencePipeline {
 input {
     File fastq1
     File fastq2
```
[the remark this answers is not in the document]
Comment (yogansun) [reply providencePipeline.wdl:6]: The requirement for this project is PE

File: providencePipeline.wdl (hunk 1 above)
Comment (yogansun) [reply providencePipeline.wdl:236]: thanks Andre, I will fix that

File: info_for_rmarkdown.pl (hunk 2 above)
Comment (yogansun) [reply info_for_rmarkdown.pl:1]: done

**Review by yogansun — APPROVED** [review]

**Review by yogansun — APPROVED** [review]

**Comment by yogansun** [comment]
please retest this

**Pull request merged**